advocacy_docs/edb-postgres-ai/analytics/reference/queries.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Queries
+description: Supported queries in Lakehouse and best practices when composing them
+---
+
+In general, **READ ONLY** queries are supported. You cannot write directly to
+object storage. This includes all Postgres built-in functions, statements
+and types. It also includes any of those provided by EPAS or PGE, depending on
+which distribution you choose to deploy.
+
+In general, you cannot insert, update, delete or otherwise modify data. You
+cannot `CREATE TABLE`. You must load data into the bucket out-of-band, either
+with your own ETL scripts or with Lakehouse Sync.  See "Advanced: Bring Your Own
+Data" for more details. (In the future, we will be making this more usable with
+a custom DDL).
```

Comment:
"(In the future, we will be making this more usable with a custom DDL)."

That's done now :) so can remove this